### ISSUE TITLE
How to share Host file system with pods in a Windows Docker Desktop configuration

### ISSUE THREAD

**Issue by arnoldfranz** [kubernetes-sigs/kind#1898]
Hello,
I am testing Kind on a Windows 10 box with Docker Desktop. The result is really awesome, Kind is an amazing software.
The target I pursue is to share files between the pods containers and the Windows 10 Host file system.
There are two main mechanisms to share file system as far as I understand:
- Docker Desktop can share file system between the W10 host and the Hyper-V Docker virtual machine
- Kind can [expose container file system](https://kind.sigs.k8s.io/docs/user/configuration/#extra-mounts) to it's host (volume mounting), therefore on the the Hyper-V Docker VM file system.

The questions:
- Is my understanding correct?
- How could I connect both, so that I could share file between W10 and the pod container? One solution would be to make a folder in the Docker Hyper-V VM that is both shared by W10 FS and the Container. The problem is that I don't master where Docker  Hyper-V VM mounts the host share.
Any idea/advice is welcome.
AF

**Comment by arnoldfranz** [comment]
Hi, 
I have found a solution.
1. In Docker Desktop / settings / resources / file sharing, define a sharing folder of your w10 box

2. Define a folder to be mapper in Kind as described in the [documentation](https://kind.sigs.k8s.io/docs/user/configuration/#extra-mounts). I named it hostfsshare

3. In kind, define a persistent volume (local) that you map on hostfsshare

4. For your application containers, create a PVC which use the PV defined above

And you will see the files created in w10 appearing in the file system of your application container.
It works great :-)

HTH
AF
